types/setting.go

```diff
@@ -891,6 +931,9 @@ var (
 		Required: true,
 		ReadOnly: false,
 		Default:  "300",
+		ValueIntRange: map[string]int{
+			ValueIntRangeMinimumKey: 0,
```

Comment:
It will re-download the BackingImage immediately when all the BackingImage on the disk is in error state  

ref: https://github.com/longhorn/longhorn-manager/pull/1029/files#diff-59782221a3f8c591d66ab9f73614211f67aba44b4ad4a31329f875f3ff0eeb25

Reply:
For all these values, I use the same definition `minimum: 0` as original check

```
	case SettingNameBackingImageCleanupWaitInterval:
		fallthrough
	case SettingNameBackingImageRecoveryWaitInterval:
		fallthrough
	case SettingNameReplicaReplenishmentWaitInterval:
		fallthrough
	case SettingNameConcurrentReplicaRebuildPerNodeLimit:
		fallthrough
	case SettingNameConcurrentBackupRestorePerNodeLimit:
		fallthrough
	case SettingNameConcurrentAutomaticEngineUpgradePerNodeLimit:
		fallthrough
	case SettingNameSupportBundleFailedHistoryLimit:
		fallthrough
	case SettingNameBackupstorePollInterval:
		fallthrough
	case SettingNameRecurringSuccessfulJobsHistoryLimit:
		fallthrough
	case SettingNameRecurringFailedJobsHistoryLimit:
		fallthrough
	case SettingNameFailedBackupTTL:
		fallthrough
	case SettingNameV2DataEngineHugepageLimit:
		value, err := strconv.Atoi(value)
		if err != nil {
			errors.Wrapf(err, "value %v is not a number", value)
		}
		if value < 0 {
			return fmt.Errorf("the value %v shouldn't be less than 0", value)
		}
```